src/server/saved_objects/migrations/core/document_migrator.ts

```diff
@@ -62,6 +62,7 @@
 
 import Boom from 'boom';
 import _ from 'lodash';
+import cloneDeep from 'lodash.clonedeep';
```

Comment:
I don't think we need this import. You should just be able to `_.cloneDeep` and things should work.